benefits/core/templates/core/page.html

```diff
@@ -54,7 +54,8 @@ <h1>{{ page.content_title }}</h1>
 
             {% block buttons %}
             {% if page.buttons|length_is:"0" %}{% else %}
-                <div class="buttons">
+                <h2 style="margin-top:20vh; font-weight:bold">{{page.choose_provider}}</h2>
+                <div style="display:flex; justify-content: space-between" class="buttons">
```

Comment:
Feel free to create your own new classes and styles in `styles.css` as necessary. We prefer styles in `styles.css` over inline styles like this.

Reply:
I'll also add, doing this here in the `page.html` template re-styles every button in the app, and not just the homepage buttons.

I think the right approach is to create a new `index.html` template that `{% extends "core/page.html" %}` and overrides the rendering of buttons for this page specifically.